### ISSUE TITLE
Infty categories

### ISSUE THREAD

**Issue by mauroporta** [mauroporta/Autour_de_DAG#10]
I'll use freely the naif idea of (infty,1) category throughout the chapter of model category as motivation for several constructions (starting from the homotopy category). The point is that classical motivations (such as "we want to invert arrows") doesn't make any sense or, at the best, sound much less convincing than the post-2008 motivations.

It will be therefore to give a detailed naif account of (\infty,1) categories in the introductions.

Does anyone have comments?

**Comment by pbelmans** [comment]
Yes. Don't. Say at most something like "once we have seen (some models of) (oo,1)-categories we gain access to other ways of motivating these concepts". Exposé 1 is model categories and 50 pages long already. Keep it at model categories, trim down the things that can be found in Hovey, and refer to Lurie for these motivations. Between 1968 and 2008 "inverting arrows" was enough motivation for studying model categories, so I guess these remain valid.

**Comment by mauroporta** [comment]
Well, we'll see what Gabriele and Bruno say about that. Moreover, I fear
its length will be dramatically cut out, as I plan to remove much of the
Reedy stuff, as it is essentially irrelevant for the notes. On the
contrary, the point of view of \infty,1 categories should be thought as
motivating point of view for the whole project, hence I really feel that to
explain better *our *motivations and referring to all the classical
literature for the others is a better thing to do.

2013/12/13 Pieter Belmans notifications@github.com

> Yes. Don't. Say at most something like "once we have seen (some models of)
> (oo,1)-categories we gain access to other ways of motivating these
> concepts". Exposé 1 is model categories and 50 pages long already. Keep it
> at model categories, trim down the things that can be found in Hovey, and
> refer to Lurie for these motivations. Between 1968 and 2008 "inverting
> arrows" was enough motivation for studying model categories, so I guess
> these remain valid.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/mauroporta/Autour_de_DAG/issues/10#issuecomment-30516950
> .

## 

To do a geometry you do not need a space, you only need an algebra of
functions on this would-be space - A. Grothendieck

**Comment by pbelmans** [comment]
But if the beginning becomes too abstract, people won't bother reading it as it will no longer be a leisurely introduction to the subject. Refer back to some things in the later exposés, but be careful about imposing too much abstraction from the beginning. It's pretty, and once you know the subject, it's nice. But it's similar to the situation of a textbook like Liu (or to a lesser extent Hartshorne) and EGA. We wish to write the first I presume, not the latter.

**Comment by mauroporta** [comment]
I never said it will be formal. But saying stuff like "the homotopy
category is introduced because one wants to invert weak equivalences"
sounds much less convincing to me than "the homotopy category is a basic
invariant obtained by cutting out all the higher homotopies, exactly as one
might expect to be done in an (\infty,1)-category".

2013/12/13 Pieter Belmans notifications@github.com

> But if the beginning becomes too abstract, people won't bother reading it
> as it will no longer be a leisurely introduction to the subject. Refer back
> to some things in the later exposés, but be careful about imposing too much
> abstraction from the beginning. It's pretty, and once you know the subject,
> it's nice. But it's comparing a textbook like Liu (or to a lesser extent
> Hartshorne) to EGA. We wish to write the first I presume, not the latter.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/mauroporta/Autour_de_DAG/issues/10#issuecomment-30519022
> .

## 

To do a geometry you do not need a space, you only need an algebra of
functions on this would-be space - A. Grothendieck

**Comment by adeel** [comment]
Hi Pieter and Mauro,

Sorry for butting in, but for some reason or other I seem to have subscribed to this repository at some point, so I "overheard" this conversation.  I just wanted to mention that I agree strongly with Pieter.  Wanting certain morphisms to become invertible is a very natural thing, in fact one might say one of the most ubiquitous patterns in modern mathematics, e.g. homological algebra (quasi-isomorphisms), topology (homotopy equivalences), category theory (equivalences), etc.  The whole point of a model structure is to make your localization into something reasonable and well-behaved.  The role of higher categories is rather orthogonal, because you want to keep track of the higher homotopies, which completely disappear in the localization of course.  So this strikes me as rather backwards, considering the historical context.  Basically, I think it would be analogous to telling a child that we want to put a group structure on the set of real numbers, to motivate them to learn addition.

Best,
Adeel\

**Comment by mauroporta** [comment]
While it is clear to me why we want to invert quasi-isomorphisms (e.g. to get functoriality of resolutions), it is a  complete mystery the fact that we want to invert homotopy equivalences in topology. Why should we care? I guess a possible answer is to get functorial CW approximation, but I am less satisfied with this answer than in the case of complexes. And what is the purpose of inverting equivalences in category theory?

In any case, I feel bad giving these as starting motivations. And I feel even worse to say "to invert arrows is a common pattern in modern mathematics". It seems to me like to say that we study localizations of rings because it is a very natural generalization of the quotient field construction! It is undoubtely true, but - come on! - we can also give a better (intuitive) explanation (which takes into account also the name of the technique): we want to study the algebraic analogue of taking an open in an affine scheme or of taking the germs at some point (even though there's no complete correspondence here, these are the geometrical reasons).

It is true that it is less elementary, but if I am writing a book on schemes, I can use the intuitive idea of scheme to motivate constructions which otherwise would be mysterious. I think to all the readers which don't want to spend months waiting for an understanding of things: if I can explain them the way I think stuff (as long as it is meaningful), it is also my duty (as a teacher) to do that. Otherwise we would be writing a Bourbaki-style stuff, and I firmly oppose myself to do that.

Moreover, the \infty categorical point of view gives motivations for many other constructions, such as mapping spaces: why should I care for hom enriched in spaces only from the point of view of model categories? Indeed the purpose of an enriched hom is to encode morphisms and homotopies between them, but this is a higher categorical point of view.

**Comment by ghost** [comment]
Hi everyone!

Sorry for my late participation!
I really agree with Adeel and Pieter on this:
I think inverting quasi-equivalences is the most important motivation for
infinity-categories (apart from the direction considered by Grothendieck in
the Pursuing stacks). A little bit everywhere in mathematics you have a
theory whose objects are "toy models" for some more fundamental theory that
you cannot define by hand (example - homotopy types: essentially, if you
are not doing analysis or geometry you don't really care about a space but
only about the homotopy type that it encloses). As we have no way to define
these more fundamentals objects by hand we have to consider bigger toy
models (in this case, topological spaces) and eliminate information
(meaning, inverting weak-equivalences).. This happens a little bit
everwhere. The problem is that you have several ways to eliminate this
extra information) and accidentaly, the best way happens to be the form of
an infinity-category.
Of course, I agree with you that you can give other self-contained
motivations for the theory of infinity-categories but if the process of
inverting equivalences would not provide an infinity-category we would not
care so much about this self-contained motivations. It would still be
interesting but just another mathematical theory. I think.

What do you think?

Best!
Marco

**Comment by mauroporta** [comment]
@Marco: I can see you point and I like the motivation passing through the homotopy types, even though I do prefer to think to homotopy types as a Bousfield localization rather than as their homotopy category. But I think this is a really good point, thanks for pointing it out! I'll add it in the motivational part about the homotopy category, if you agree!

In any case, the motivations for \infty-categories I was thinking were something like: we want to classify objects not only up-to-isomorphisms, but only up to weak equivalences, like in the case of Perf. Then one is naturally led to consider higher coherence relations between quasi-isomorphisms, which can be thought as higher homotopies. This happens to be what we would like to think as an (\infty,1)-category. Another motivation could be the Kontsevich hidden smoothness principle: singular moduli spaces are shadows of non-singular ones, and a reasoning involving the tangent complex shows that the affines should be taken to be cdga. Then, to glue in a meaningful way such objects one has to consider higher coherence conditions again, so he is led to model categories (hence to (\infty,1)-categories).

Those motivations seem to me more concrete and more geometrical than "we want to invert arrows because it is an interesting process" and more convincing that "we want to invert arrows to get functorial resolutions (for chain complexes)". In a sense, I would say that if following these intuitions we wouldn't end up with \infty categories, then we wouldn't care about \inty categories as much as we do, especially in the setting of the book.

What does anyone else think?

Anyway, I'm not even sure I have been able to explain exactly what I was planning to do. So I think I'll finish to write the motivational parts and then I'll ask you to read them and to give me a feedback.

**Comment by pbelmans** [comment]
What you say is of course true. But consider it from the viewpoint of someone who doesn't know what derived algebraic geometry is, and takes a look at the table of contents or leitfaden of the book. He or she wishes to get a feel for the geometry as soon as possible. In the current setup this would mean he or she has to read exposés 1, 6--8. By keeping these four exposés as down-to-earth as possible we provide a means to this. Afterwards one understand the need for (oo,1)-categories (at least in my case it went this way), and can go back to get a nice introduction to these. 

Hence, I'm still not convinced that exposé 1 (or the notes in general, for that matter) is the best place for this to go.

If you write these things: do it in a different branch. That's what branches are for, and that way everyone can see what's happening with respect to this.

**Comment by mauroporta** [comment]
Ok, I'll do that in a different branch.

2013/12/14 Pieter Belmans notifications@github.com

> What you say is of course true. But consider it from the viewpoint of
> someone who doesn't know what derived algebraic geometry is, and takes a
> look at the table of contents or leitfaden of the book. He or she wishes to
> get a feel for the geometry as soon as possible. In the current setup this
> would mean he or she has to read exposés 1, 6--8. By keeping these four
> exposés as down-to-earth as possible we provide a means to this. Afterwards
> one understand the need for (oo,1)-categories (at least in my case it went
> this way), and can go back to get a nice introduction to these.
> 
> Hence, I'm still not convinced that exposé 1 (or the notes in general, for
> that matter) is the best place for this to go.
> 
> If you write these things: do it in a different branch. That's what
> branches are for, and that way everyone can see what's happening with
> respect to this.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/mauroporta/Autour_de_DAG/issues/10#issuecomment-30568125
> .

## 

To do a geometry you do not need a space, you only need an algebra of
functions on this would-be space - A. Grothendieck

**Comment by mauroporta** [comment]
I uploaded a new version, if you want to read the introduction to the homotopy category and give me your feedback, it would be useful!

Ah, the pseudocircle should be obviously written better than this... but I'll do that another day!

**Comment by adeel** [comment]
Sorry for the late reply.

> In any case, the motivations for \infty-categories I was thinking were something like: we want to classify objects not only up-to-isomorphisms, but only up to weak equivalences

But this is exactly the point of localization.  If you have something defined up to something weaker than isomorphism, then the right setting for this object is in the homotopy category.

> it is a  complete mystery the fact that we want to invert homotopy equivalences in topology. Why should we care?

I know very little about topology, but even in higher category theory it is necessary to look at the homotopy category of spaces.  For example, mapping spaces in an $(\infty,1)$-category are defined only up to weak equivalence, i.e. they are only well-defined in Ho(SSet); this is discussed already in the first chapter of HTT.  More generally, I guess the point of homotopy theory is to consider topological spaces only up to homotopy equivalence, so in other words homotopy theory is the study of the localization of Top or SSet at the class of homotopy equivalences.  As far as I understand, the original motivation for Quillen's theory of model categories is to give a good description of Ho(Top) and also to give a precise meaning to the general feeling that the homotopy theories of topological spaces and simplicial sets are the same.

> And what is the purpose of inverting equivalences in category theory?

It is well-known that is "wrong" to consider categories up to strict isomorphism.  Therefore one normally works in Ho(Cat), which happens to be identified with the category of small categories and isomorphism classes of functors.  Indeed there is a model structure on Cat where two morphisms are homotopic iff they are isomorphic as functors.  Similarly for dg-categories, one normally works in the localization with respect to quasi-equivalences, or in noncommutative geometry, in the localization with respect to Morita equivalences.  Both these localizations are described by model structures, a fact which is underlying most of the usefulness of dg-categories in commutative or noncommutative geometry.

On the other hand, part of the problem with triangulated categories, in my view, is that there is no model structure where the weak equivalences are triangulated equivalences (as far as I know).  This is probably the underlying reason for the difficulty in getting a nice statement about Fourier-Mukai transforms at the triangulated level, though everything works out nicely at the dg level.  In case you are not familiar with Fourier-Mukai functors, let me just say that Orlov has proved that, for nice enough projective schemes, every fully faithful triangulated functor between the derived categories D(X) \to D(Y) is induced by some complex E \in D(X \times Y), in the sense that is isomorphic to the functor which pulls back to the product, tensors with E, and pushes forward again.  Since the 90's this has been expected to be true for all triangulated functors, but so far no one can prove this.  But on the dg level this was settled by Toen in his paper on homotopy theory of dg-categories.

In summary, this is how I see things: first of all, you want to talk about localizations like D(A) = Ho(C(A)), Ho(Top), Ho(Cat), Ho(DGCat), etc.; a priori these things are not well-behaved, so you consider model structures; and finally model categories turn out to be just a convenient way of presenting (infty,1)-categories.

**Comment by mauroporta** [comment]
Thank you. In fact Marco already pointed out what you said about Ho(Top), and I already agreed with that, and now the construction of the category of homotopy types is included in the motivational parts.

What I was saying is that I don't like motivations such as "let's do this because it is a generalization of something else", and in my vision the classification problem takes directly to the (\infty,1)-world. The homotopy category is the naif attempt to do things, but since so many informations are lost, we need really soon additional data such as homotopy limits, homotopy colimits and so on. Hence, I feel like the homotopy category is a little bit aged, and classical motivations for that should be integrated with a more modern point of view.

In any case, it is pointless to continue this discussion, I already uploaded a proposal. If you want to read it and give me a feedback, it will be useful!
